storagetransfer/storagetransfer_test.go

```diff
@@ -16,21 +16,18 @@ package storagetransfer
 
 import (
 	"bytes"
+	"cloud.google.com/go/iam"
```

Comment:
Imports should be separated by language builtin vs. other dependencies (which will have a full import URL). Intellij should handle this correctly for you I think.

Reply:
`goimports -w`